File: 1011/main.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let raio = parseFloat(lines.shift());
let pi = 3.14159;
let volume = pi * (4/3.0) * Math.pow(raio,3);

console.log(`VOLUME = ${volume.toFixed(3)}`);

File: 1011/main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner input = new Scanner(System.in);
        
        double raio, volume, pi = 3.14159;
        
        raio = input.nextDouble();
        
        volume = (4.0/3) * pi * Math.pow(raio, 3);
        
        System.out.println(String.format("VOLUME = %.3f", volume));
 
    }
 
}